pfcpiface/upf.go

```diff
@@ -45,28 +45,40 @@ type pdr struct {
 	srcIface     uint8
 	tunnelIP4Dst uint32
 	eNBTeid      uint32
-	ueAddrIP     uint32
-	inetIP       uint32
-	uePort       uint16
-	inetPort     uint16
+	srcIP        uint32
+	dstIP        uint32
+	srcPort      uint16
+	dstPort      uint16
 	proto        uint8
 
 	srcIfaceMask     uint8
 	tunnelIP4DstMask uint32
 	eNBTeidMask      uint32
-	ueAddrIPMask     uint32
-	inetIPMask       uint32
-	uePortMask       uint16
-	inetPortMask     uint16
+	srcIPMask        uint32
+	dstIPMask        uint32
+	srcPortMask      uint16
+	dstPortMask      uint16
 	protoMask        uint8
```

Comment:
how many of these need to be controlled from outside vs can be declared as consts? Can the whole `Masks:` member below be a const?